fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -72,8 +72,12 @@ def command(message)
     def command_output(message)
       actual = (message.split("\r").last || "") # as clearing the line will remove the `>` and the time stamp
       actual.split("\n").each do |msg|
-        prefix = msg.include?("▸") ? "" : "▸ "
-        log.info(prefix + "" + msg.magenta)
+        if FastlaneCore::Env.truthy?("FASTLANE_DISABLE_OUTPUT_FORMAT")
+          log.info(msg)
```

Comment:
@pepibumur 

> Do you think it makes sense to group the two flags or should we use a new flag for this?

Grouping the two flags means using `log.info(msg.magenta)` instead?

If so, only disabling prefix looks better to me 👍 
And it would be better rename `FASTLANE_DISABLE_OUTPUT_FORMAT` to `FASTLANE_DISABLE_COMMAND_OUTPUT_PREFIX` or something.

Reply:
Thinking about it I think disabling any kind of formatting/prefixing makes more sense. Instead of adding flags for each individual element, it's just one flag that disables everything.